makefile

```diff
@@ -13,10 +13,14 @@ build: vet osx linux
 	go install .
 
 test: build
-	big inventory add marathon
-	big up -d marathon
+	docker pull python:3
```

Comment:
It speeds up the test runs themselves so the mesos-agent doesn't have to pull the docker image.